pinot/regressors/biophysical_regressor.py

```diff
@@ -23,95 +24,103 @@ class BiophysicalRegressor(torch.nn.Module):
 
     """
 
-    def __init__(self, base_regressor=None, *args, **kwargs):
+    def __init__(self, base_regressor_class=None, *args, **kwargs):
         super(BiophysicalRegressor, self).__init__()
-        self.base_regressor = base_regressor
+        # get the base regressor
+        self.base_regressor_class = base_regressor_class
+        self.base_regressor = base_regressor_class(
+                *args, **kwargs
+        )
+
+        # initialize measurement parameter
         self.log_sigma_measurement = torch.nn.Parameter(torch.zeros(1))
 
-    def g(self, func_value=None, test_ligand_concentration=1e-3):
-        return 1 / (1 + torch.exp(-func_value) / test_ligand_concentration)
+    def _get_measurement(self, delta_g, concentration=1e-3):
+        """ Translate ..math:: \Delta G to percentage inhibition.
+
+        Parameters
+        ----------
+        delta_g : torch.Tensor, shape=(number_of_graphs, 1)
+           Binding free energy. 
+
+        concentration : torch.Tensor, shape=(,) or (1, number_of_concentrations)
+            Concentration of ligand.
+
+        Returns
+        -------
+        measurement : torch.Tensor, shape=(number_of_graphs, 1)
```

Comment:
I think there's a typo in this equation.

If you want _percent_ inhibition, you'll want something like
```python
100 * (1 - 1.0 / (1.0 + concentration / torch.exp(delta_g))
```
where `concentration` must be in Molar units.

This means that when `delta_g << 0` (a tight binder), we have 100% inhibition, and when `delta_g ~ 0`, we should have 0% inhibition.

Reply:
So what I did was copy the function g from Josh's notebook. 
Concentration is indeed assumed to be in Molar units in this code.

Let me check out how your comments map to that code so I understand your comment better.

Reply:
Also, while we're at it, I only ever got Josh's notebook as a reference for this function. Where can I look up the published version of this so I can check the math independently? I should have asked for this earlier to debug my own code, I rushed this too much.